rasa_nlu/data_router.py

```diff
@@ -35,6 +54,15 @@ def __init__(self, config, component_builder):
         self.emulator = self.__create_emulator()
         self.component_builder = component_builder if component_builder else ComponentBuilder(use_cache=True)
         self.model_store = self.__create_model_store()
+        self.pool = ProcessPool(config['max_training_processes'] or 1)
```

Comment:
No value means that the server will start as many processes as it has CPU cores. I think 1 a good default value regarding the server load: each training process is already parallelized (joblib on which sklearn relies use multiprocessing for parallel computation). So most of your CPU power and memory space will already be eaten by one process. But nothing prevent the user to specify a higher value. Or were you expressing concerns about the syntax? Is that so ugly? 😝